spec_helper.rb

```diff
@@ -39,3 +39,7 @@
 end
 
 $VERBOSE = nil unless ENV['OUTPUT_WARNINGS']
+
+if PlatformGuard.windows?
+  MSpec.disable_feature(:fork)
+end
```

Comment:
I think it would be better to have this in https://github.com/ruby/spec/blob/master/default.mspec.
Explicit `enable`/`disable` is fine, so sth like
if windows; disable; else; enable; end